wikimedia/wikipedia-ios
# Project changes for Xcode 14

**Phabricator:** 
https://phabricator.wikimedia.org/T310091

### Notes
These are project changes needed for moving to Xcode 14. Each commit message is fairly descriptive - I'm happy to chime in if there's any questions during PR review. I also spun off a couple of follow up tasks based on some of these warnings - [T317901](https://phabricator.wikimedia.org/T317901) & [T317905](https://phabricator.wikimedia.org/T317905).

### Test Steps
1. Run app on a few devices, some with iOS 16, some with lower OSes if possible.
2. Confirm there are no build warnings.

Build server has been set up with Xcode 14, but we'll test deployment with future builds and fix any issues in a follow up PR.
